build.ps1

```diff
@@ -364,18 +362,20 @@ Remove-Item variable:global:_MSBuildExe -ea Ignore
 # Import Arcade
 . "$PSScriptRoot/eng/common/tools.ps1"
 
+# Add default .binlog location if not already on the command line. tools.ps1 does not handle this; it just checks
+# $BinaryLog, $CI and $ExcludeCIBinarylog values for an error case. But tools.ps1 provides a nice function to help.
+if ($BinaryLog) {
+    $bl = GetMSBuildBinaryLogCommandLineArgument($MSBuildArguments)
+    if (-not $bl) {
+        $MSBuildArguments += "/bl:" + (Join-Path $LogDir "Build.binlog")
+    }
+}
+
 # Capture MSBuild crash logs
 $env:MSBUILDDEBUGPATH = $LogDir
 
 $local:exit_code = $null
 try {
-    # Import custom tools configuration, if present in the repo.
-    # Note: Import in global scope so that the script set top-level variables without qualification.
-    $configureToolsetScript = Join-Path $EngRoot "configure-toolset.ps1"
-    if (Test-Path $configureToolsetScript) {
-      . $configureToolsetScript
-    }
-
```

Comment:
This was old cruft. Sourcing tools.ps1 sources this file for us already and makes all settings available in this scope. Same for the similar block near the end of build.sh.